docs/OV_Runtime_UG/model_representation.md

```diff
@@ -0,0 +1,95 @@
+
+For details on how to build an OpenVINO Model, see the [Build Model from source](@ref build_model) section.
+
+## Operations
+
+`ov::Op` represents any abstract operations in the OpenVINO Model representation. You need to use this class to create [custom operations](../OV_Runtime_UG/Extensibility_DG/AddingNGraphOps.md).
+
+## Operation Sets
+
+Operation set represents the set of some OpenVINO operations. `ov::OpSet` is a class which provide a functionality to work with operation sets.
+OpenVINO provides separate namespace for each operation set.
+
+## Static and Partial Shapes
+
+OpenVINO has two types for shape representation: 
+
+* `ov::Shape` - Represents static (fully defined) shapes.
+
+* `ov::PartialShape` - Represents dynamic shapes. That means that the rank or some of dimensions are dynamic (undefined). `ov::PartialShape` can be converted to `ov::Shape` using the `get_shape()` method if all dimensions are static; otherwise the conversion raises an exception.
+
+
+## Build OpenVINO Model from source {#build_model}
+
+OpenVINO Model can be created from source, this section illustrates how to construct an OpenVINO Model composed of operations from an available opset.
+
+Operation Set `opsetX` integrates a list of OpenVINO pre-compiled operations that work for this purpose. In other words, `opsetX` defines a set of operations for building a graph.
+
+For a complete list of operation sets supported by OpenVINO, see [Available Operations Sets](../ops/opset.md).
+
+To add suport of custom operations, see the [Add Custom OpenVINO Operations](../OV_Runtime_UG/Extensibility_DG/Intro.md) document.
+
+To build `ov::Model` from `opset8` operations please include next files:
+
+ - C++
+      @snippet example_ngraph_utils.cpp ov:include
+
+ - Python
+     ```python
+      import openvino.runtime.opset8 as ov
+      from openvino.runtime import Model
+      ```
+
+Below an example of simple OpenVINO Model
+
+ - C++
+      @snippet example_ngraph_utils.cpp ov:create_simple_model
+
+ - Python
+     ```python
+     TBD
+      ```
+
+
+Example of OpenVINO Model with several outputs
+
+ - C++
+      @snippet example_ngraph_utils.cpp ov:create_advanced_model
+
+ - Python
+     ```python
+     TBD
+      ```
+
+## FAQ
+
+ - Does OpenVINO have any capabilities to debug the Model structure and Model modification?
+   - To receive additional messages about applied graph modifications, rebuild the OpenVINO runtime library with the `-DENABLE_OPENVINO_DEBUG=ON` option.
+   - OpenVINO Model can be visualized to image from xDot format:
+    @snippet example_ngraph_utils.cpp ov:visualize
+   - OpenVINO Model can be serialized to IR:
+    @snippet example_ngraph_utils.cpp ov:serialize
+ - How can I develop my own transformation pass?
+   - Please take a look to [Transformations Developer Guide](./nGraphTransformation.md)
+
+## See Also
+
+* [Available Operation Sets](../ops/opset.md)
+* [opset1 Specification](../ops/opset1.md)
```

Comment:
Consider removing opset1, opset2 .... from the list - they are available through [Available Operation Sets](../ops/opset.md)